docs/reference/search/rrf.asciidoc

```diff
@@ -4,8 +4,8 @@
 preview::["This functionality is in technical preview and may be changed or removed in a future release. The syntax will likely change before GA. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
 
 https://plg.uwaterloo.ca/~gvcormac/cormacksigir09-rrf.pdf[Reciprocal rank fusion (RRF)]
-is a method for combining multiple result sets with different relevance
-indicators into a single result set. RRF requires no tuning, and the different
+is a method for combining multiple top documents sets with different relevance
```

Comment:
Are we actively trying to change our vocabulary from `result` to `top documents`? It may be unintuitive or confusing?

Reply:
I was doing this since the notion of a retriever is an entity that returns top documents. So I wanted to try to be consistent with that. I think I will change this one back to results, though, since it's not in the context of a retriever here.